src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -44,12 +45,12 @@ private void Awake()
      */
     public void LoadSequences()
     {
-        if (Directory.Exists(SEQUENCES_PATH))
+        if (Directory.Exists(_sequencePath))
```

Comment:
2 things
1. You can't use the Directory API to load Resources in the built game binary.
2. Somewhat related to 1 ... we need to build the set of the directory + the resources when running in the built binary.  The same name 'may' appear in both in which case we should prefer loading the one from the directory, but there will also be some that have not been overridden that only appear as an in game resource.

Reply:
More context from slack conversation...


```
for a single sequence loading...
Editor - Loads from the Assets folder using File APIs
Runtime - Loads from the persistent data path (if said file exists), else falls back to loading as a resource - effectively treating the persistent data path as an override of any compiled in sequences


So... For finding all the sequences that exist...
Editor - Use file apis on the Assets folder path (you have this part)
Runtime - Needs to find all the sequences in the persistent data path (you have this part)
                   PLUS - also needs to find all sequences that are resources (you do NOT have this part) ... and only load those that aren't overridden (same file name and sequence name inside json) in the persistent data path
```